api/admin/routes.py

```diff
@@ -803,6 +936,58 @@ def patron_auth_self_tests(identifier):
 @requires_admin
 @requires_csrf_token
 def lookup_patron():
+    """Look up personal information about a patron via the ILS
```

Comment:
The `200` response code says "A confirmation that the password was updated" which doesn't match this description. Could you clarify specifically what this does? The fact that it's a POST makes me feel like it is actually probably doing something like resetting a password.